crates/consensus/src/receipt/receipts.rs

```diff
@@ -102,6 +103,45 @@ impl<T> From<ReceiptWithBloom<T>> for Receipt<T> {
     }
 }
 
+/// A collection of receipts organized as a two-dimensional vector.
+#[derive(
+    Clone, Debug, PartialEq, Eq, Default, From, derive_more::Deref, DerefMut, IntoIterator,
+)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+pub struct Receipts {
```

Comment:
I can see this being helpful, but this should be generic over the internal type

Reply:
Indeed better, just fixed